### ISSUE TITLE
Godot editor crashes when trying to open a .os script, even an empty script.

### ISSUE THREAD

**Issue by Gnumaru** [Vahera/godot-orchestrator#407]
### Describe the bug

When trying to open a .os script, even a newly created one, the editor will crash. Even running the project directly with --verbose will give absolutely no error message. The log file for the project wil also be almost emtpy, with only the content bellow.

```
2024-06-15 21:44:12 [INFO]: Starting Orchestrator v2.0.1.stable.official
2024-06-15 21:44:12 [INFO]: Using Godot Engine v4.2.2.stable.official
2024-06-15 21:44:13 [INFO]: Initializing OrchestratorScript
```

Changing the renderer between compatibility, mobile or forward+ makes no difference.

### Expected behavior

The script should open successfully in the orchestrator main window.

### Actual behavior

The editor crashes silently.

### How to Reproduce?

1) Open the attached project
2) Install orchestrator version 2.0.1.stable
3) Try to open an existing .os script or creating and opening a new one

### Godot full version

v4.2.2.stable.official [15073afe3]

### Orchestrator version

2.0.1.stable **AND** v2.1.dev2.official

### Additional information

I'm using a X86_64 build of godot on a windows 10 X86_64 installation. The system info is the one bellow:

Godot v4.2.2.stable - Windows 10.0.19045 - GLES3 (Compatibility) - NVIDIA GeForce GTX 1060 6GB (NVIDIA; 31.0.15.3623) - AMD A10-7850K Radeon R7, 12 Compute Cores 4C+8G (4 Threads)

I'll also put a sample project here. I had to remove the addon before compressing otherwise the zip file would be bigger than 25MB and github would not let me attach it here.

[G4OrchestratorTest.zip](https://github.com/user-attachments/files/15855809/G4OrchestratorTest.zip)

Just open it, install the 2.0.1.stable version of orchestrator and try to open the "node_2d.os" script, the way you open it does not matter, double clicking it, opening a scene and clicking in the script icon or opening the orchestrator main view and trying to open the script, all of them crash.

There was one time I got an error, but it may not be related to the problem

```
ERROR: FATAL: Index p_index = -1 is out of bounds (size() = 0).
   at: godot::CowData<struct OrchestratorMainView::ScriptFile>::get (D:\a\godot-orchestrator\godot-orchestrator\extern\godot-cpp\include\godot_cpp/templates/cowdata.hpp:170)
```

![image](https://github.com/Vahera/godot-orchestrator/assets/5783414/7faf3642-4351-4270-8a00-3ecaebf7b7e6)

**Comment by Gnumaru** [comment]
O tried the workaround mentioned here and it worked, the editor stopped crashing and I could open up a .os script.

https://github.com/Vahera/godot-orchestrator/issues/335

Then I tested more and noticed I didn't need the workaround, the steam version of godot just worked whereas the version downloaded from godotengine.org didn't.

Then I tested more and realized that the version downloaded from godotengine.org would work If I turned it into a self contained version, that is, if I created an empty file named ._sc_ in the godot executable directory.
